bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TVITourBookItem.java

```diff
@@ -22,31 +22,34 @@
 import java.util.ArrayList;
 import java.util.HashSet;
 
+import net.tourbook.application.TourbookPlugin;
 import net.tourbook.common.UI;
 import net.tourbook.common.time.TimeTools;
 import net.tourbook.common.time.TourDateTime;
 import net.tourbook.common.util.TreeViewerItem;
 import net.tourbook.data.TourData;
 import net.tourbook.database.TourDatabase;
+import net.tourbook.preferences.ITourbookPreferences;
 import net.tourbook.tour.ITourItem;
 import net.tourbook.tour.TourManager;
 
+import org.eclipse.jface.preference.IPreferenceStore;
+
 public abstract class TVITourBookItem extends TreeViewerItem implements ITourItem {
 
    static ZonedDateTime       calendar8 = ZonedDateTime.now().with(TimeTools.calendarWeek.dayOfWeek(), 1);
-
    /**
     * All tour fields in the tourbook view, the first field is <code>tourId</code> which can be
     * prefixed with <code>DISTINCT</code>
     */
    public static final String SQL_ALL_TOUR_FIELDS;
 
    public static final String SQL_ALL_OTHER_FIELDS;
-   public static final int    SQL_ALL_OTHER_FIELDS__COLUMN_START_NUMBER;
 
+   public static final int    SQL_ALL_OTHER_FIELDS__COLUMN_START_NUMBER;
    static final String        SQL_SUM_COLUMNS;
-   static final String        SQL_SUM_FIELDS;
 
+   static final String        SQL_SUM_FIELDS;
```

Comment:
code is reformatted

Reply:
Arg, Eclipse !! I will fix that

Reply:
Fixed in my current [PR](https://github.com/wolfgang-ch/mytourbook/pull/233)